source/documentation/rules-g.html.md.erb

```diff
@@ -0,0 +1,1208 @@
+
+<b>Functional Description</b>
+
+The Reference number shall include the ARC number or the fallback eAD reference number when the ‘Type’ of the ‘Additional reference’ is C651 or C658 respectively.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0057
+
+<b>Functional Description</b>
+
+Common Code List can be extended or restricted at national level. Purely national codes are not included in Common Domain messages.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0058
+
+<b>Functional Description</b>
+
+When /*/Consignment/HouseConsignment/ConsignmentItem/PreviousDocument/type is in SET {C651, C658} the Unique Body Reference (UBR) is required to be recorded in this field.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0061
+
+<b>Functional Description</b>
+
+The information presented in in this D.G. is related to Safety & Security and to the Binding Itinerary. In case of Binding itinerary, the information entered must include the list of codes of the countries between the Office of Departure and the Office of Destination. If more information is available about the countries visited by the means of transport since it's first place of loading until the last place of unloading, it should also be added for Safety & Security purpose only.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0062
+
+<b>Functional Description</b>
+
+The rules R0506 and R0507 are applied on CC015C and CC013C to ensure that the declaration does not include unnecessary and repetitive information. They must be enforced by all NTA. Considering the possibility that one Goods Item is taken out from the declaration during the control, the message CC029C and CD001C may have different content from CC015C (or CC013C or CC170C). Consequently, those rules R0506 and R0507 shall not be strictly enforced on the Common Domain messages. Certainly not by the recipient of the CD message, likely not by the sender of the CD message.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0064
+
+<b>Functional Description</b>
+
+The message must be sent within thirteen (13) days from the day the “Destination Control Results” C_DES_CON (IE018) message is sent to the Office of Departure
+
+<b>Technical Description</b>
+
+N/A
+
+##G0068
+
+<b>Functional Description</b>
+
+The Data Group &lt;CONSIGNMENT- HOUSE CONSIGNMENT- CONSIGNMENT ITEM- ADDITIONAL REFERENCE&gt; will be also used to include the information of EMCS consignment exported from one EU member state into a Non-EU-Member state, in case of Export Followed by Transit (where in messages CC013C or CC015C the &lt;CONSIGNMENT- HOUSE CONSIGNMENT- PREVIOUS DOCUMENT.Type&gt; = ‘N830’ AND &lt;CONSIGNMENT- HOUSE CONSIGNMENT- CONSIGNMENT ITEM- ADDITIONAL REFERENCE. Type&gt; is in SET CL234 (DocumentTypeExcise)).<br/>In this case, the Data Group &lt;GOODS SHIPMENT- GOODS ITEM- PREVIOUS DOCUMENT&gt; of the Export declaration, will be mapped with the Data Group &lt;CONSIGNMENT- HOUSE CONSIGNMENT- CONSIGNMENT ITEM- ADDITIONAL REFERENCE&gt; of the Transit declaration.   
+
```

Comment:
The first sentence is too long and need to be split. Suggest inserting a full stop after `state` and changing `in case of Export` to `If Export is`.